dao/mongodb.rya/src/test/java/org/apache/rya/mongodb/instance/MongoRyaDetailsRepositoryIT.java

```diff
@@ -61,20 +59,14 @@
 
     @BeforeClass
     public static void startMiniAccumulo() throws MongoException, IOException {
```

Comment:
There's a lot of other accumulo references in this file that should be renamed too.